insight/src/test/java/info/nightscout/androidaps/TestBase.kt

```diff
@@ -0,0 +1,39 @@
+package info.nightscout.androidaps
+
+import info.nightscout.rx.AapsSchedulers
+import info.nightscout.rx.TestAapsSchedulers
+import info.nightscout.rx.logging.AAPSLoggerTest
+import org.junit.Before
+import org.junit.Rule
+import org.mockito.Mockito
+import org.mockito.junit.MockitoJUnit
+import org.mockito.junit.MockitoRule
+import java.util.*
+
+open class TestBase {
+
+    val aapsLogger = AAPSLoggerTest()
+    val aapsSchedulers: AapsSchedulers = TestAapsSchedulers()
+
+    // Add a JUnit rule that will setup the @Mock annotated vars and log.
+    // Another possibility would be to add `MockitoAnnotations.initMocks(this) to the setup method.
+    @get:Rule
+    val mockitoRule: MockitoRule = MockitoJUnit.rule()
```

Comment:
This has to be changed for junit5. See other modules

Reply:
Done